bindings/java/Makefile

```diff
@@ -30,11 +30,10 @@ test: build
 release:
 	$(call build_command,cd src/bc && swig -java -package bc -outcurrentdir ../../../c/include/bc.i)
 	$(call build_command,javac src/bc/*.java)
-	$(call build_command,gcc -c -O3 src/bc/bc_wrap.c -I$(JAVAINCLUDE) -I../c/include/ -o src/bc/bc_wrap.o)
-	$(call build_command,gcc -shared -O3 src/bc/bc_wrap.o ../../target/release/deps/libbattlecode.a $(LIBS) -o $(TARGET))
+	$(call build_command,gcc -c -O3 -fPIC src/bc/bc_wrap.c -I$(JAVAINCLUDE) -I../c/include/ -o src/bc/bc_wrap.o)
```

Comment:
make sure this code is the same as in `build`, you'll have an extra `-I` here

Reply:
Nicely spotted